src/MemoryTracing/MemoryInfoProducer.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "MemoryTracing/MemoryInfoProducer.h"
+
+#include <chrono>
+#include <thread>
+
+#include "MemoryTracingUtils.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/ThreadUtils.h"
+#include "capture.pb.h"
+
+namespace orbit_memory_tracing {
+
+using orbit_grpc_protos::SystemMemoryUsage;
+
+void MemoryInfoProducer::Start() {
+  exit_requested_ = false;
+  thread_ = new std::thread(&MemoryInfoProducer::Run, this);
```

Comment:
As mentioned below, never use `new` and `delete` (unless there's a very specific reason like Qt ownership and such).

Reply:
Changed to use `unique_ptr` and `reset`. Also thanks for telling me about this! I will pay attention in the future!